cnxeasybake/scripts/main.py

```diff
@@ -20,7 +20,8 @@ def easybake(css_in, html_in=sys.stdin, html_out=sys.stdout, last_step=None,
     oven.bake(html_doc, last_step)
 
     # serialize out HTML
-    print(etree.tostring(html_doc, method="xml"), file=html_out)
+    print(etree.tostring(html_doc, method="xml").decode('utf-8'),
+          file=html_out)
```

Comment:
Can you pass `etree.tostring(..., encoding='utf-8')` and get similar results?

Reply:
I tested this in python 3 and no, it doesn't change the type of the output, but it changes the way unicode characters are represented:

```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from lxml import etree
>>> root = etree.Element('root')
>>> root.text = 'Każda'
>>> etree.tostring(root, method='xml')
b'<root>Ka&#380;da</root>'
>>> etree.tostring(root, method='xml', encoding='utf-8')
b'<root>Ka\xc5\xbcda</root>'
```

Reply:
O... I think we actually want the later non-xml-encoded results. I'm basing that on feedback I received from Helene when making adjustments to the cnx-epub squash-to-text functionality.

You can see that specific change at https://github.com/openstax/cnx-epub/pull/146/commits/d7080a7bb3007e9bac921d76cbc010dbeefbacf9.

Perhaps this is realm for a wider discussion though. Because if it's currently encoding xml entities, we should probably leave it doing that.